packages/api-client/src/views/Settings/SettingsGeneral.vue

```diff
@@ -195,6 +202,44 @@ const changeTheme = (themeId: ThemeId) => {
                 </div>
               </ScalarButton>
             </div>
+            <div class="mt-4 mb-2">
+              <p class="text-c-1 font-bold leading-[21px">Integration Themes</p>
+            </div>
+            <div class="grid grid-cols-2 gap-2">
+              <ScalarButton
+                v-for="themeId in integrationThemeIds"
+                :key="themeId"
+                class="px-2"
+                :class="[
+                  'flex items-center justify-between gap-2 text-c-1 border-1/2',
+                  activeWorkspace.themeId === themeId ? 'bg-b-2' : 'bg-b-1',
+                ]"
```

Comment:
```suggestion
                class="px-2 flex items-center justify-between gap-2 text-c-1 border-1/2"
                :class="
                  activeWorkspace.themeId === themeId ? 'bg-b-2' : 'bg-b-1',
                "
```
keep the static classes in `class`